Components/Inputs/inputs.css

```diff
@@ -0,0 +1,75 @@
+@import url("../../default.css");
+
+/* form and input styles */
+.form-container {
+  width: fit-content;
+  display: flex;
+  flex-direction: column;
+}
+
+.form-group {
+  padding: 0rem 2rem 2rem;
+  display: flex;
+  flex-direction: column;
+  background-color: var(--light-color);
+  box-shadow: rgba(0, 0, 0, 0.16) 0px 3px 6px, rgba(0, 0, 0, 0.23) 0px 3px 6px;
+}
+
+.form-group button {
+  width: 10rem;
+}
+
+input,
+textarea {
+  display: inline-block;
```

Comment:
If one block of code used many times just make it utility class for easy user experience acc. To OOCSS paradigm

Reply:
Okay rushi. I'll add them